packages/kbn-es-query/src/es_query/__tests__/handle_nested_filter.js

```diff
@@ -0,0 +1,84 @@
+/*
+ * Licensed to Elasticsearch B.V. under one or more contributor
+ * license agreements. See the NOTICE file distributed with
+ * this work for additional information regarding copyright
+ * ownership. Elasticsearch B.V. licenses this file to you under
+ * the Apache License, Version 2.0 (the "License"); you may
+ * not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+
+import { handleNestedFilter } from '../handle_nested_filter';
+import indexPattern from '../../__fixtures__/index_pattern_response.json';
+import { buildPhraseFilter, buildQueryFilter } from '../../filters';
+import expect from '@kbn/expect';
+
+describe('handleNestedFilter', function () {
+
+  it('should return the filter\'s query wrapped in nested query if the target field is nested', () => {
+    const field = getField('nestedField.child');
+    const filter = buildPhraseFilter(field, 'foo', indexPattern);
+    const result = handleNestedFilter(filter, indexPattern);
+    console.log(JSON.stringify(result, null, 2));
```

Comment:
`console.log`?

Reply:
whoops, thanks, I'll get that outta there :)